components/google_drive/sources/new-or-modified-files/new-or-modified-files.mjs

```diff
@@ -20,11 +20,25 @@ export default {
   key: "google_drive-new-or-modified-files",
   name: "New or Modified Files",
   description: "Emit new event any time any file in your linked Google Drive is added, modified, or deleted",
-  version: "0.1.0",
+  version: "0.1.1",
   type: "source",
   // Dedupe events based on the "x-goog-message-number" header for the target channel:
   // https://developers.google.com/drive/api/v3/push#making-watch-requests
   dedupe: "unique",
+  hooks: {
+    async deploy() {
+      console.log("Deploying...");
```

Comment:
```suggestion
      console.log("Deploying...");
```

Reply:
Changed to console.log("Deploying..."); to follow Google's Developer docs style guide.